tests/test_extractors.py

```diff
@@ -80,3 +92,53 @@ async def test_async_emulators() -> None:
         assert len(all_emu_mps_results[True].processed_data) == len(
             all_emu_mps_results[False].processed_data
         )
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("extractor", [RemoteQPUExtractor, RemoteEmuMPSExtractor])
+async def test_async_remote_qpu_extractor(extractor: Type[BaseRemoteExtractor]) -> None:
+    """
+    Test that the remote extractors can execute without exploding (both sync and async).
+    """
+    # Load dataset
+    original_ptcfm_data = [
+        cast(pyg_data.Data, d)
+        for d in PygRetrier().insist(pyg_dataset.TUDataset, root="dataset", name="PTC_FM")
+    ]
+    MAX_NUMBER_OF_SAMPLES = 5
+
+    ptcfm_compiler = PTCFMCompiler()
+
+    fake_value = "fake-value"
```

Comment:
What's that?

Reply:
I renamed this to `placeholder` and added a comment, let me know if that's ok for you